modules/retail/pages/containerized-saltboot.adoc

```diff
@@ -0,0 +1,158 @@
+[[retail-install-setup-containerized]]
+= Set Up the {productname} {smr} Environment using containerized proxy
+
+To set up the {productname} {smr} environment, you will need to have already installed and configured:
+
+* {productname} {smr} Server 4.3 or newer
+* one or more {productname} {smr} containerized proxies
+* one or more {productname} build host
+
+This section covers how to configure your  environment using containerized {productname} Proxy for [systemitem]``Saltboot`` deployment.
+
+[NOTE]
+====
+Containerized {productname} {smr} environment does not use {productname} {smr} Branch Servers.
+====
+
+[IMPORTANT]
+====
+Containerized workflow requires POS images build using {productname} Server 4.3 or newer.
+Older images will not work.
+====
+
+[IMPORTANT]
+====
+Containerized workflow not longer implicitly configure DHCP for PXE booting.
+See xref:specialized-guides:salt/salt-formula-dhcpd.adoc[] how to use DHCP formula to configure DHCP server.
+
+Without using DHCP formula, make sure your DHCP server has correct PXE booting setting pointing to containerized proxy.
+See below for example.
+====
+
+== Saltboot group
+
+Containerized {productname} {smr} is configured in [systemitem]``Saltboot Group``.
+Saltboot Groups are regular server groups with [systemitem]``Saltboot Group`` formula enabled.
+
+Saltboot Groups acts as a branch groups.
+
+image::saltboot_group.png[scaledwidth=80%]
+
+[systemitem]``Saltboot Group`` formula is a successor of [systemitem]``Branch Network formula``, [systemitem]``PXE formula`` and [systemitem]``TFTP formula`` used in regular {productname} {smr} setups.
+
+Name of the [systemitem]``Saltboot group`` is automatically used as a [systemitem]``branch id``, an identifier for group of machines booted through particular containerized {productname} Proxy.
+
+All [systemitem]``saltboot`` deployed machines though containerized proxy will automatically became members of its [systemitem]``Saltboot group``.
+
+To connect [systemitem]``Saltboot group`` with containerized proxy fill [systemitem]``Image Download Server`` entry with Fully Qualified Domain Name ([literal]``FQDN``) of the containerized proxy.
+
+The rest of configuration is optional.
+
+=== Default boot image
+
+Configure [systemitem]``Default boot image for new registrations`` to specify what boot image should be booted by not yet registered POS terminal.
+This is useful when stable boot image is wanted for initial deployments.
+Without this setting, newest built boot image is used as default boot image.
+
+If [systemitem]``Default boot image for new registrations`` is set, option to set its version appears under name [systemitem]``Default boot image version`` where specific image version can be set.
+
+=== Kernel option for the saltboot group
+
+Option [systemitem]``Kernel parameters for the group`` can be used to pass extra kernel options to all POS terminals registered withing this saltboot group.
```

Comment:
We should add a comment here to troubleshooting the issue I add, solved by included `id=dhcp`